.vscode/settings.json

```diff
@@ -1,22 +1,22 @@
 {
     "workbench.colorCustomizations": {
-        "activityBar.activeBackground": "#1f6fd0",
-        "activityBar.activeBorder": "#ee90bb",
-        "activityBar.background": "#1f6fd0",
-        "activityBar.foreground": "#e7e7e7",
-        "activityBar.inactiveForeground": "#e7e7e799",
-        "activityBarBadge.background": "#ee90bb",
+        "activityBar.activeBackground": "#ff6433",
```

Comment:
- [ ] Hello @team, I am suggesting that this file is added to `.gitignore` because it is not in any way related to the current project.